query-engine/query-engine/src/server/mod.rs

```diff
@@ -55,9 +56,14 @@ pub async fn listen(opts: PrismaOpt) -> PrismaResult<()> {
         .await?;
 
     let mut app = tide::with_state(State::new(cx, opts.enable_playground, opts.enable_debug_mode));
+
+    if opts.enable_playground {}
+
     app.with(ElapsedMiddleware::new());
 
-    app.at("/").post(graphql_handler);
+    if opts.enable_playground {
+        app.at("/").with(TimingMiddleware::new()).post(graphql_handler);
+    }
```

Comment:
I think that means that the `graphql_handler` will only be available if the playground is enabled.

Reply:
You're right this isn't good. Fixed it in a subsequent patch!